### DISCUSSION TITLE
Where can i find example with 2 microservices with Kafka communication

### DISCUSSION THREAD

**Discussion by P9avel** [abpframework/abp#11992] [General]
Hi to all. I am need example with 2 microservices with Kafka communication. Please send me link to example

**Comment by maliming** [comment]
You can check this microservices project https://github.com/abpframework/eShopOnAbp
but it may not use Kafka.

**Reply by P9avel** [reply]
Thank you for reply. But i am new ABP's user and eShopOnAbp very simple for me. Possible can you to prepare example with docker compose and 2 microservices and postgress and kafka

**Reply by maliming** [reply]
No plan for now.
